### PR TITLE
Editorial fix for #2479

### PR DESCRIPTION
**Pull request by ianswett**
I noticed this earlier today when trying to resolve conflicts relating to the connection ID change.

### CODE REVIEW

File: draft-ietf-quic-invariants.md [hunk 1]
```diff
@@ -161,21 +161,18 @@ All other bits in that byte are version specific.
...
-value 0xe0 describes a 17 byte Destination Connection ID and a zero byte Source
-Connection ID.
-
-The connection ID lengths are followed by two connection IDs.  The connection
-ID associated with the recipient of the packet (the Destination Connection ID)
-is followed by the connection ID associated with the sender of the packet (the
-Source Connection ID).
+The byte contains the length in bytes of the Destination Connection ID (see
+{{connection-id}}) field that follows it.  This length is encoded as an 8-bit
+unsigned integer.  The Destination Connection ID field follows the DCI Len
```
Comment (martinthomson) [inline draft-ietf-quic-invariants.md]:
```suggestion
unsigned integer.  The Destination Connection ID field follows the DCID Len
```

File: draft-ietf-quic-invariants.md [hunk 2]
```diff
@@ -161,21 +161,18 @@ All other bits in that byte are version specific.
...
+and is between 0 and 255 bytes in length.
```
Comment (martinthomson) [inline draft-ietf-quic-invariants.md]:
```suggestion
field and is between 0 and 255 bytes in length.
```

File: draft-ietf-quic-invariants.md [hunk 3]
```diff
@@ -161,21 +161,18 @@ All other bits in that byte are version specific.
...
+
+The byte following the Source Connection ID contains the length in bytes
+of the Source Connection ID field that follows it.  This length is encoded as
+a 8-bit unsigned integer.  The Source Connection ID field follows the SCID Len
+and is between 0 and 255 bytes in length.
```
Comment (martinthomson) [inline draft-ietf-quic-invariants.md]:
```suggestion
field and is between 0 and 255 bytes in length.
```

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -3503,16 +3503,13 @@ Version:
...
-  high bits of the byte and the Source Connection ID Length (SCIL) field
-  occupies the 4 low bits of the byte.  An encoded length of 0 indicates that
-  the connection ID is also 0 bytes in length.  Non-zero encoded lengths are
-  increased by 3 to get the full length of the connection ID, producing a length
-  between 4 and 18 bytes inclusive.  For example, a byte with the value 0x50
-  describes an 8-byte Destination Connection ID and a zero-length Source
-  Connection ID.
+: The byte following the version contains the length in bytes of the Destination
+  Connection ID field that follows it.  This length is encoded as an 8-bit
+  unsigned integer.  In QUIC version 1, this value MUST NOT exceed 20 bytes.
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md]:
```suggestion
  unsigned integer.  In QUIC version 1, this value MUST NOT exceed 20.
```

Using "bytes" here, while correct, implies that the length of this field is what is being limited.

**Comment by ianswett** [comment]
Suggestions accepted.

File: draft-ietf-quic-invariants.md [hunk 5]
```diff
@@ -161,21 +161,18 @@ All other bits in that byte are version specific.
...
+{{connection-id}}) field that follows it.  This length is encoded as an 8-bit
+unsigned integer.  The Destination Connection ID field follows the DCID Len
+field and is between 0 and 255 bytes in length.
+
+The byte following the Source Connection ID contains the length in bytes
+of the Source Connection ID field that follows it.  This length is encoded as
+a 8-bit unsigned integer.  The Source Connection ID field follows the SCID Len
+field and is between 0 and 255 bytes in length.
+
+{{negotiating-connection-ids}} describes the use of the source and destination
```
Comment (martinthomson) [inline draft-ietf-quic-invariants.md]: This is what is causing CI to fail.  This anchor is in a different document.

File: draft-ietf-quic-invariants.md [hunk 6]
```diff
@@ -161,21 +161,15 @@ All other bits in that byte are version specific.
...
-the full length of the connection ID; the final value is therefore either 0 or
-between 4 and 18 bytes in length (inclusive).  For example, a byte with the
-value 0xe0 describes a 17 byte Destination Connection ID and a zero byte Source
-Connection ID.
-
-The connection ID lengths are followed by two connection IDs.  The connection
-ID associated with the recipient of the packet (the Destination Connection ID)
-is followed by the connection ID associated with the sender of the packet (the
-Source Connection ID).
+The byte contains the length in bytes of the Destination Connection ID (see
```
Comment (janaiyengar) [inline draft-ietf-quic-invariants.md]:
```suggestion
The next byte contains the length in bytes of the Destination Connection ID (see
```

File: draft-ietf-quic-invariants.md [hunk 7]
```diff
@@ -161,21 +161,15 @@ All other bits in that byte are version specific.
...
+{{connection-id}}) field that follows it.  This length is encoded as an 8-bit
+unsigned integer.  The Destination Connection ID field follows the DCID Len
+field and is between 0 and 255 bytes in length.
+
+The byte following the Source Connection ID contains the length in bytes
```
Comment (janaiyengar) [inline draft-ietf-quic-invariants.md]:
```suggestion
The next byte contains the length in bytes
```

**Pull request merged**